### PR TITLE
move test-zeekoe.py to dev folder

### PR DESCRIPTION
**Pull request by DariusParvin**
and update paths within test to work with new directory

### CODE REVIEW

File: dev/test-zeekoe.py [hunk 1]
```diff
@@ -121,39 +121,39 @@ def run_command(cmd, verbose):
 
 def start_merchant_server(merch_config, verbose):
     info("Starting the merchant server...")
-    cmd = ["./target/debug/zkchannel", "merchant", "--config", merch_config, "run"]
+    cmd = ["../target/debug/zkchannel", "merchant", "--config", merch_config, "run"]
```
Comment (yaymukund) [inline dev/test-zeekoe.py:124]: Could you make this a constant at the top of the file?

File: dev/test-zeekoe.py [hunk 2]
```diff
@@ -263,7 +263,7 @@ def run_command_list(self, command_list):
 def main():
     parser = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter)
     parser.add_argument("command", help="", nargs="?", default="list")
-    parser.add_argument("--path", help="path to create configs", default="./dev")
+    parser.add_argument("--path", help="path to create configs", default=".")
```
Comment (yaymukund) [inline dev/test-zeekoe.py:266]: Non-blocking: While you're touching this code, could this be something a bit clearer like `--config-path`?

File: dev/test-zeekoe.py (hunk 1 above)
Comment (DariusParvin) [reply dev/test-zeekoe.py]: done

File: dev/test-zeekoe.py [hunk 3]
```diff
@@ -263,7 +265,7 @@ def run_command_list(self, command_list):
 def main():
     parser = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter)
     parser.add_argument("command", help="", nargs="?", default="list")
-    parser.add_argument("--path", help="path to create configs", default=".")
+    parser.add_argument("--config-path", help="path to create configs", default=".")
```
Comment (yaymukund) [inline dev/test-zeekoe.py:268]: The usage also needs to be updated on L292

**Review by yaymukund — CHANGES_REQUESTED** [review]

File: dev/test-zeekoe.py (hunk 3 above)
Comment (DariusParvin) [reply dev/test-zeekoe.py:268]: oh yes, good catch. I've renamed `dev_path` to `config_path` everywhere else too.

**Pull request merged**